docs/rfcs/rfc-007-subscriptions.md

```diff
@@ -276,6 +276,19 @@ UNWIND meta AS m
 RETURN collect(DISTINCT m) AS meta
 ```
 
+##### Examples of subscriptions to relationships
+Get the source and target nodes of a relationship in subscription:
+
+
+ ```graphql   
+ personConnected {  // Non-directed, another event will be triggered for movies
+    to { Movie | Actor } // Target node, union is the list of all types related to a person
+    from { Actor }
```

Comment:
An initial thought I had following yesterday's conversation is that `to` and `from` imply direction. May we be better with `connectedNode` and perhaps the name of the node, `person`?